CHANGES.rst

```diff
@@ -5,6 +5,7 @@ Version 0.9.1 (UNRELEASED)
 --------------------------
 
 - Administrators:
+    - Adds configuration environment variable ``reana_server.environment.REANA_INTERACTIVE_SESSION_MAX_INACTIVITY_PERIOD`` to set a limit in days for the maximum inactivity period of interactive sessions after which they will be closed.
```

Comment:
I think this can be skipped, it is just an internal change so I feel like there is no need to add it to the CHANGES. In any case, CHANGES.rst are checked and fixed before any stable release.